api/nakadi-event-bus-api.yaml

```diff
@@ -623,6 +623,50 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  '/event-types/{name}/cursors':
+      get:
+        tags:
+          - unmanaged-api
+          - monitoring
+          - management-api
+        security:
+          - oauth2: ['nakadi.event_stream.read']
+        description: |
+          Transforms a list of Cursors with `shift` into a list without `shift`s. This is useful when there is the need
+          for randomly access events in the stream.
+
+        parameters:
+          - name: name
+            in: path
+            description: EventType name
+            type: string
+            required: true
+          - name: X-Flow-Id
+            in: header
+            description: |
+              The flow id of the request, which is written into the logs and passed to called services. Helpful
+              for operational troubleshooting and log analysis.
+            type: string
+          - name: X-nakadi-cursors
+            in: header
```

Comment:
@dehora we are working on providing a clear guidance. We would be please to work out the guidance together with the team.

@dehora @rcillo @codecov-io @v-stepanov @lmontrieux Please contribute to the discussion in https://github.com/zalando/restful-api-guidelines/issues/207 how the guidance should look like.